### ISSUE TITLE
Migrate to Gulp 4

### ISSUE THREAD

**Issue by Acid5000** [romannurik/AndroidAssetStudio#225]
The code is not compiled. New Gulp released.  And the old one, as luck would have it, requires some kind of dependency that is outdated or deleted.

https://zzz.buzz/2016/11/19/gulp-4-0-upgrade-guide/

**Comment by romannurik** [comment]
Sorry are you suggesting that we migrate to Gulp 4? That's certainly possible, but is more a feature request than a bug.

**Comment by nick-homecu** [comment]
@romannurik 
It may be helpful to a lot of people to migrate to Gulp 4.

I cloned the repo
Ran `npm install`
Tried to run `npm run build` and `npm run start`

Both resulted in an error 
Failed to load external module @babel/register

I attempted to use that, but it didn't help which led me to this answer
https://stackoverflow.com/a/55926692/9996381

It seems that people with a node version of 12 and above can't build the project?

I'm going to try an older version of node, but hopefully to help save time, what version of node or you running @romannurik?
